src/rust/engine/process_executor/Cargo.toml

```diff
@@ -16,10 +16,12 @@ grpc_util = { path = "../grpc_util" }
 hashing = { path = "../hashing" }
 log = "0.4"
 process_execution = { path = "../process_execution" }
+docker = { path = "../process_execution/docker"}
```

Comment:
`process_executor` doesn't currently use the Docker `CommandRunner` (although it might eventually...) so you should be able to remove this.
```suggestion
```